### DISCUSSION TITLE
What's the latest version of the branch deployment and prod deployment Github Actions for Dagster serverless?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17637] [Q&A]
Can someone point me to the `latest` version of the Branch Deployment and Prod deployment Github Actions of Dagster Serverless?

The reason I asked is that I found <https://github.com/dagster-io/dagster-cloud-cicd-action|this repo> and it seems like it’s being deprecated in favor of the ones in <https://github.com/dagster-io/dagster-cloud-action/|this repo>. So that’s fine.
&gt; This Action is being phased out in favor of our <https://github.com/dagster-io/dagster-cloud-action/|new set of GitHub Actions>, for more information see our <https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/|Hybrid> and <https://github.com/dagster-io/dagster-cloud-serverless-quickstart/|Serverless> quickstart repos.
But then if you look at the <https://github.com/dagster-io/dagster-cloud-serverless-quickstart/blob/main/.github/workflows/branch_deployments.yml|dagster-cloud-serverless-quickstart> repo, it doesn’t have the same version of branch deployment and prod deploy <https://github.com/dagster-io/dagster-cloud-action/blob/main/.github/workflows/serverless_branch_deployments.yml|as this one> but has been updated more recently (4 months instead of 1 year).

TLDR; Which repo as the most updated Github Actions script for Dagster Serverless<https://github.com/dagster-io/dagster-cloud-serverless-quickstart/blob/main/.github/workflows/branch_deployments.yml| option 1> or <https://github.com/dagster-io/dagster-cloud-action/blob/main/.github/workflows/serverless_branch_deployments.yml|option 2.>

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C03SMUY0SFQ/p1698935799981799?thread_ts=1698935799.981799&cid=C03SMUY0SFQ).*

**Comment by shalabhc** [comment]
Sorry for the confusion here. The latest version is actually here: https://github.com/dagster-io/dagster-cloud-action/tree/main/github/serverless. You can replace the `${{ secrets.DAGSTER_CLOUD_URL }}` with the literal value for your org (eg `"https://ORG.dagster.cloud"`) to make it easier to debug since secrets are never shown in Github. 

This should also match what's in `dagster-cloud-serverless-quickstart` which shows sample user code with the Github action.

We will add some comments to these workflows and remove obsolete ones to make it clear where to get the latest version. Note that if you use the Dagster Cloud UI to set up a serverless repository, it will automatically copy the latest version of the workflows into your repo.
